api/src/main/java/org/pmiops/workbench/db/model/DbDataset.java

```diff
@@ -277,101 +260,4 @@ public int hashCode() {
   public String toString() {
     return ToStringBuilder.reflectionToString(this);
   }
-
-  public static DbDataset.Builder builder() {
```

Comment:
I had to delete this otherwise DataSetMapper was using this method for the method  DbDataset dataSetRequestToDb(DataSetRequest dataSetRequest); 
and then starting throwing exception (for each attribute)  **"lastModifiedTIME" not found did you mean null**